### PR TITLE
notification

### PR DESCRIPTION
**Pull request by ThanhSonnTL**
## Related Tickets
- [#Task xxxxx: Content](https://edu-redmine.sun-asterisk.vn/issues/xxxx)
## WHAT
- Notification
## Evidence 
a280bccda3547b0a2245
(Screenshot or Video)
e32dd86fc7f61fa846e7
9339.jpg)

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/blob/master/eng/android/coding_convention_android_kotlin.md | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/sun/android/notification/NotificationActivity.kt [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        ) {
+            val notificationChannel = NotificationChannel(
+                CHANNEL_ID,
+                "Mascot Notification", NotificationManager.IMPORTANCE_HIGH
+            )
+            with(notificationChannel) {
+                enableLights(true)
+                lightColor = Color.RED
+                enableVibration(true)
+                description = "Notification from Mascot"
```
Comment (ghost) [inline app/src/main/java/com/sun/android/notification/NotificationActivity.kt]: string resource

File: app/src/main/java/com/sun/android/notification/NotificationActivity.kt [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    }
+
+    private fun updateNotification() {
+        val androidImage: Bitmap = BitmapFactory
+            .decodeResource(resources, R.drawable.donut_circle)
+        val notifyBuilder = getNotificationBuilder()
+        notifyBuilder.setStyle(
+            NotificationCompat.BigPictureStyle()
+                .bigPicture(androidImage)
+                .setBigContentTitle("Notification Updated!")
```
Comment (ghost) [inline app/src/main/java/com/sun/android/notification/NotificationActivity.kt]: .

File: app/src/main/java/com/sun/android/notification/NotificationActivity.kt [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        val notificationIntent = Intent(this, MainActivity::class.java)
+
+        val notificationPendingIntent = PendingIntent.getActivity(
+            this,
+            NOTIFICATION_ID, notificationIntent, PendingIntent.FLAG_UPDATE_CURRENT
+        )
+
+
+        val notifyBuilder = NotificationCompat.Builder(this, CHANNEL_ID).apply {
+            this.setContentTitle("You've been notified!")
```
Comment (ghost) [inline app/src/main/java/com/sun/android/notification/NotificationActivity.kt]: .

Comment (minhnn-1279) [inline app/src/main/java/com/sun/android/notification/NotificationActivity.kt]: extract string resource

**Pull request merged**